content/03.results.md

```diff
@@ -2,41 +2,48 @@
 
 ### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas
 
-We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Symposium (PNOC) (**Figure {@fig:Fig1}A**). 
-We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
-We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or the open-access CAVATICA project (See **Data and code availability** section).
-As new analytical products (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses were produced, or issues with the data were identified, new releases were made available in a versioned manner.
+We previously (**TODO: looks like this "previously" was only pHGG?**) performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**). 
+We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses including variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
+A summarized view of the number of biospecimens per phase of therapy across different histology groupings is shown in (**Figure {@fig:Fig1}B**). 
+We maintained a data release folder on Amazon S3, downloadable using the GitHub repository or the open-access [CAVATICA project](https://www.cavatica.org/), containing merged files for each analysis (See **Data and code availability** section). **TODO: this is mentioned before mention the repo itself!**
```

Comment:
The TODO here is organizational. We can either talk about the repo earlier, or not talk about the repo in this sentence at all.

Reply:
I'm rephrasing this to remove a reference to the repo itself, and say you can download it directly from S3 itself. The repo contains a script that does just that, so the overall meaning should be the same.